doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -54,12 +54,12 @@ be cast to the type of the command being queried.
 
 1::
 The returned function pointer must: only be called with a dispatchable object
-(the first parameter) that is a child of pname:instance.
+(the first parameter) that is sname:VkInstance or a child of pname:instance.
     e.g. sname:VkInstance, sname:VkPhysicalDevice, sname:VkDevice, sname:VkQueue, or
     sname:VkCommandBuffer.
 
 2::
-available extension is an extension function supported by any of the loader, ICD or layer.
+An "available extension" is an extension function supported by any of the loader, ICD or layer.
```

Comment:
That looks like a glossary term definition, in which case it should be marked up with underscores _available extension_ and have a matching entry in the Glossary appendix.